src/main/steps/respondent/yourdocuments/alldocuments/drug_alcohol_tests/content.ts

```diff
@@ -49,7 +49,10 @@ export const generateContent: TranslationFn = content => {
   const translations = languages[content.language]();
   const drugCitizenDocs: object[] = [];
   for (const doc of content.userCase?.citizenUploadedDocumentList || []) {
-    if (doc.value.documentType === 'Drug and alcohol tests (toxicology)') {
+    if (
+      doc.value.isApplicant === content.byApplicant &&
+      doc.value.documentType === 'Drug and alcohol tests (toxicology)'
```

Comment:
Very fragile condition.

Reply:
Yeah done this with idamId and doc type.